libs/api/observability/src/lib/trace-wrappers/resolver-trace-wrapper.ts

```diff
@@ -0,0 +1,63 @@
+import { ModulesContainer } from '@nestjs/core';
+import { InstanceWrapper } from '@nestjs/core/injector/instance-wrapper';
+import {
+	RESOLVER_NAME_METADATA,
+	RESOLVER_TYPE_METADATA,
+} from '@nestjs/graphql';
+
+import { TraceWrapper } from './abstract-trace-wrapper';
+
+// Wraps all NestJS Resolvers `Query` and `Mutation` methods with OpenTelemetry spans
```

Comment:
```suggestion
/**
 * Wraps all NestJS Resolvers `Query` and `Mutation` methods with OpenTelemetry spans
 */
```

Shows the comment in docs as well.